tests/include_find_all_test.js

```diff
@@ -0,0 +1,219 @@
+          Price = this.sequelize.define('Price', {
+            value: DataTypes.FLOAT
+          }),
+          Customer = this.sequelize.define('Customer', {
+            name: DataTypes.STRING
+          }),
+          Group = this.sequelize.define('Group', {
+            name: DataTypes.STRING
+          }),
+          GroupMember = this.sequelize.define('GroupMember', {
+
+          }),
+          Rank = this.sequelize.define('Rank', {
+            name: DataTypes.STRING,
+            canInvite: {
+              type: DataTypes.INTEGER,
+              defaultValue: 0
+            },
+            canRemove: {
+              type: DataTypes.INTEGER,
+              defaultValue: 0
+            },
+            canPost: {
+              type: DataTypes.INTEGER,
+              defaultValue: 0
+            }
+          });
+
+        this.models = {
+          User,
+          Company,
+          Product,
+          Tag,
+          Price,
+          Customer,
+          Group,
+          GroupMember,
+          Rank
+        };
+
+        User.hasMany(Product);
+        Product.belongsTo(User);
+
+        Product.belongsToMany(Tag, { through: 'product_tag' });
+        Tag.belongsToMany(Product, { through: 'product_tag' });
+        Product.belongsTo(Tag, { as: 'Category' });
+        Product.belongsTo(Company);
+
+        Product.hasMany(Price);
+        Price.belongsTo(Product);
+
+        User.hasMany(GroupMember, { as: 'Memberships' });
+        GroupMember.belongsTo(User);
+        GroupMember.belongsTo(Rank);
+        GroupMember.belongsTo(Group);
+        Group.hasMany(GroupMember, { as: 'Memberships' });
+
+        await this.sequelize.sync({ force: true });
+        await Group.bulkCreate([
+          { name: 'Developers' },
+          { name: 'Designers' },
+          { name: 'Managers' }
+        ]);
+        const groups = await Group.findAll();
+        await Company.bulkCreate([
+          { name: 'Sequelize' },
+          { name: 'Coca Cola' },
+          { name: 'Bonanza' },
+          { name: 'NYSE' },
+          { name: 'Coshopr' }
+        ]);
+        const companies = await Company.findAll();
+        await Rank.bulkCreate([
+          { name: 'Admin', canInvite: 1, canRemove: 1, canPost: 1 },
+          { name: 'Trustee', canInvite: 1, canRemove: 0, canPost: 1 },
+          { name: 'Member', canInvite: 1, canRemove: 0, canPost: 0 }
+        ]);
+        const ranks = await Rank.findAll();
+        await Tag.bulkCreate([
+          { name: 'A' },
+          { name: 'B' },
+          { name: 'C' },
+          { name: 'D' },
+          { name: 'E' }
+        ]);
+        const tags = await Tag.findAll();
+        for (const i of [0, 1, 2, 3, 4]) {
+          const user = await User.create();
+          // Edited this part because a test expect id to be 3.
+          // This maintains the test procedure, giving Product predictable ids.
+          await Product.bulkCreate([
+            { id: i * 5 + 1, title: 'Chair' },
```

Comment:
Sorry I'm a bit confused here, why do we have the *5 + 1?

Reply:
I'm sorry I did not make it clear. It is because if repeats in an array and I want ids to be sequential. So the original idea is from this same suite, but at the `expect`. You'll see it increases id one by one, from 1 to 15.

Reply:
Right above there is an array [0, 1, 2, 3, 4]. Each iteration will multiply by 5 and add one. So:
first iteration: [1, 2, 3, 4, 5]
second iteration: [6,  7, 8, 9, 10]
third iteration: [11, 12, 13, 14, 15]
...